examples/kubernetes/kernel-check/kernel-check.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
One small comment on this type https://github.com/cilium/cilium/issues/12384#issuecomment-653876639.